Dockerfile.armhf

```diff
@@ -1,5 +0,0 @@
-FROM centurylink/ca-certs
-ENV GODEBUG=netdns=go
```

Comment:
I think we need `GODEBUG=netdns=go`

Reply:
You are right, added it back.

Reply:
What behavior exhibited the need?

I'm just curious, because it only forces the use of the Go resolver in case the user compiles it w/ CGO. https://golang.org/pkg/net/#hdr-Name_Resolution

Reply:
I have just added it back to avoid any risk of break compared to the previous version

Reply:
Fair enough for me!